charts/enterprise-logs/templates/admin-api/deployment-admin-api.yaml

```diff
@@ -42,8 +42,10 @@ spec:
       priorityClassName: {{ .Values.adminApi.priorityClassName }}
       {{- end }}
       securityContext:
-        {{- toYaml .Values.adminApi.securityContext | nindent 8 }}
+        fsGroup: 10001
```

Comment:
why do we use hardcoded value here but not variable that can be changed in values file?

Reply:
I think if we did want to make it a variable, the loki-distributed chart uses `podSecurityContext` to differentiate it from the container SecurityContexts IIRC.

Reply:
Good question, and one I asked myself as well. I decided to hard-code the value here, because it must not be changed or otherwise it breaks.
The other `securityContext`s are configurable to be as coherent with the `loki-distributed` chart as possible.

Reply:
ok, thanks.